rpistart.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash 
+
+
+set -e 
+
+
+cd "$(dirname "$0")"
+
+
+ARCH=arm64
+ROOT_DIR=/home/ubuntu
+NUM_CORES=${NUM_CORES:-$(nproc)}
+IMAGE_FILE=""
+KERNEL_VERSION="linux-6.1.34"
+
+
+# First step: Install the required packages. These include cross-compilers for arm64
+# and required packages for QEMU itself. 
+# NOTE: We probably don't need to install `qemu-system-gui`
+sudo apt install \
+        gcc-aarch64-linux-gnu \
+        g++-aarch64-linux-gnu \
+        qemu \
+        qemubuilder \
+        qemu-system-gui \
+        qemu-system-arm \
+        qemu-utils \
+        qemu-system-data \
+        qemu-system \
+        flex \
+        bison \
+        libssl-dev 
```

Comment:
libssl-dev is for being able to generate a hashed password with openssl. It turns out the password MUST be hashed.